core/where_clause.rs

```diff
@@ -439,13 +439,17 @@ fn translate_condition_expr(
             }
             _ => todo!(),
         },
-        ast::Expr::InList { lhs, not, rhs } => {}
+        ast::Expr::InList {
+            lhs: _,
+            not: _,
+            rhs: _,
+        } => {}
```

Comment:
No need to change this pull request, but for future: please don't mix bug fixes and cleanups. They make merging harder because cleaning up code increases likelihood of merge conflicts.

And don't get me wrong, we _love_ cleanups, just make them completely separate pull requests.

Reply:
sure, should have kept it separate